postcode_lookup/templates/base.html

```diff
@@ -1227,17 +1207,6 @@ <h2 class="c-breadcrumb__title" id="system-breadcrumb">
              section
             </h2>
             <ul class="u-list-reset c-breadcrumb__menu">
-             <li class="c-breadcrumb__item">
-              <a href="/">
-               <span>
-                Home
-               </span>
-              </a>
-              <svg class="o-icon" focusable="false" role="presentation">
-               <use xlink:href="#icon-chevron" xmlns:xlink="http://www.w3.org/1999/xlink">
-               </use>
-              </svg>
-             </li>
```

Comment:
Is it intentional that we've lost the "home" breadcrumb here?

**before:**

<img width="329" height="111" alt="Screenshot at 2025-08-12 10-21-59" src="https://github.com/user-attachments/assets/3702394e-3f28-4a0b-b0b9-cf5e7de00b18" />

**after:**

<img width="307" height="110" alt="Screenshot at 2025-08-12 10-21-50" src="https://github.com/user-attachments/assets/ccaff989-88a2-4b1c-abd1-93e83c60ce04" />

Reply:
Yes...

We use a donor page and all design outside our page content comes from there. They seem to have removed this on other pages in the same section, e.g https://www.electoralcommission.org.uk/voting-and-elections/ways-vote/apply-vote-proxy